sdk/helper/consts/jwt.go

```diff
@@ -38,19 +38,11 @@ var (
 		jose.HS512,
 	}
 
-	// AllowedJWTSignatureAlgorithmsOIDC defines the JWT signature algorithms
-	// allowed for OIDC operations.
-	AllowedJWTSignatureAlgorithmsOIDC = []jose.SignatureAlgorithm{
-		jose.RS256,
-		jose.RS384,
-		jose.RS512,
-		jose.ES256,
-		jose.ES384,
```

Comment:
@dc-tec `jose.ES384` is missing from the consolidated list. This is causing the current test failures as far as I can see.